src/styles/card.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  border: 1px solid rgb(243, 243, 243);
+  width: 200px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
`* {box-sizing: border-box;}`
you might put this inside " * " to reset this feature for the entire document.